app/assets/stylesheets/app/no_modal.scss

```diff
@@ -0,0 +1,5 @@
+.not-in-modal {
```

Comment:
This felt a little awkward, but it was easier to fix in CSS than in the structure of the partials.